### PR TITLE
use openssl for example migration, since it is simpler than the compiler rebuild

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@regro/auto-tick ready for review!

File: docs/migrators.rst [hunk 1]
```diff
@@ -46,51 +47,36 @@ Example of migrator addition function:
...
-    # normally we build all the cycle packages at once and let the maintianers
-    # deal with it
+    # everything which depends on openssl and has not predecessors in the 
+    # openssl dependents graph
+    top_level = {node for node in gx.successors("openssl") if
+                 (node in total_graph) and
+                 len(list(total_graph.predecessors(node))) == 0}
     cycles = list(nx.simple_cycles(total_graph))
-    print('cycles are here:', cycles)
+    # print('cycles are here:', cycles)
```
Comment (scopatz) [inline docs/migrators.rst]: Should this line just be removed?

Comment (CJ-Wright) [reply docs/migrators.rst]: done

**Comment by CJ-Wright** [comment]
thank you!@

**Pull request merged**